src/interfaces/logfiles.js

```diff
@@ -43,10 +44,20 @@ function mountApi(app) {
       res.json(files)
     })
   })
-  app.get('/logfiles/:filename', function(req, res, next) {
+  app.get(`${SERVERROUTESPREFIX}/logfiles/:filename`, function(req, res, next) {
     const sanitizedLogfile = path
       .join(getFullLogDir(app), req.params.filename)
       .replace(/\.\./g, '')
     res.sendFile(sanitizedLogfile)
   })
+  app.get(`${SERVERROUTESPREFIX}/ziplogs`, function(req, res, next) {
+    const d = new Date()
+    const zipFileName = "logs-" + d.getFullYear() + "-" + ("0"+(d.getMonth()+1)).slice(-2) + "-"
```

Comment:
please convert this simple but hard to understand logic to use `moment().format`

would it be convenient to include the vessel's name in the filename? Sanitised, so that all but alphanumerics are replaced with underscore.

Reply:
Replaced with moment()

In terms of boat name will need your help to find out how to access it  from here.

Reply:
app.config.vesselName, app.config.vesselMMSI

The filename needs sanitizing, l'd replace regexp `\W` with underscore or something like that.

Reply:
Done